phpunit/class-block-library-comment-template-test.php

```diff
@@ -137,9 +137,9 @@ function test_rendering_comment_template() {
 
 		// Here we use the function prefixed with 'gutenberg_*' because it's added
 		// in the build step.
-		$this->assertEquals(
-			'<ol ><li id="comment-' . self::$comment_ids[0] . '" class="comment even thread-even depth-1"><div class="wp-block-comment-author-name"><a rel="external nofollow ugc" href="http://example.com/author-url/" target="_self" >Test</a></div><div class="wp-block-comment-content">Hello world</div></li></ol>',
-			gutenberg_render_block_core_comment_template( null, null, $block )
+		$this->assertSame(
+			str_replace( array( "\n", "\t" ), '', '<ol ><li id="comment-' . self::$comment_ids[0] . '" class="comment even thread-even depth-1"><div class="wp-block-comment-author-name"><a rel="external nofollow ugc" href="http://example.com/author-url/" target="_self" >Test</a></div><div class="wp-block-comment-content"><p>Hello world</p></div></li></ol>' ),
+			str_replace( array( "\n", "\t" ), '', gutenberg_render_block_core_comment_template( null, null, $block ) )
```

Comment:
Real talk: Even though I wrote it, I don't know how I feel about this. `autop` puts in some weird line breaks that I want to ignore but a line break is white space rather than an empty string.